viewing_party/main.py

```diff
@@ -0,0 +1,172 @@
+def create_movie(title, genre, rating):
+    movie_dic = {}
+    if title and genre and rating:
+        movie_dic["title"] = title
+        movie_dic["genre"] = genre
+        movie_dic["rating"] = rating
+        return movie_dic
+    else:
+        return None
+
+
+
+def add_to_watched(user_data,movie):
+        user_data["watched"].append(movie)
+        return user_data
+
+
+def add_to_watchlist(user_data, movie):
+        user_data["watchlist"].append(movie)
+        return user_data
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+
+    for movie in watchlist:
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            watchlist.remove(movie)
+            
+
+    return user_data
+            
+        
+# print(wave 2 *********************************)
+
+def get_watched_avg_rating(user_data):
+    watched_list = user_data["watched"]
+    sum = 0
+    average = 0
+    
+    if len(watched_list) == 0:
+        return average
+
+    for movie in watched_list:
+        sum += movie["rating"]
+        average = sum /len(watched_list)
+    return average
+
+def get_most_watched_genre(user_data):
+    most_watched = user_data["watched"]
+
+# what gender is most frequently occurring in the watched list.
+# return genre
+#  count how many times each genre shows up. counts of each genre.
+    popular_genre = {}
+
+    for movie in most_watched:
+        genre = movie["genre"]
+        if genre in  popular_genre:
+            popular_genre[genre] += 1
+
+        else:
+            popular_genre[genre] = 1
+
+    max = 0
+    max_genre = None
+    for genre, count in popular_genre.items():
+        if count > max:
+            max = count
+            max_genre = genre
+        
+
+
+            
+    return max_genre
+
+
+# # print(wave3 ***************************) 
+def make_friends_list(user_data):
+    friends_list = []
+    for friend_movie in user_data["friends"]: 
+        for movie in friend_movie["watched"]:
+           
+#  some nesting to pull out the friends movies
+                friends_list.append(movie)
```

Comment:
You should indent your comments to match the code around them:

```suggestion
                #  some nesting to pull out the friends movies
                friends_list.append(movie)
```